api/config/rolepermission.php

```diff
@@ -501,11 +545,22 @@
             ],
```

Comment:
The `update-team-team` permission is no longer associated with this role according to the spreadsheet.